frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -194,6 +226,82 @@ const updateItemAppConnectTo = (
   return k8sPatch(model, item, patch);
 };
 
+export const createServiceBinding = (
+  source: K8sResourceKind,
+  target: K8sResourceKind,
+): Promise<any> => {
+  if (!source || !target || source === target) {
+    return Promise.reject();
+  }
+
+  const targetName = _.get(target, 'metadata.name');
+  const sourceName = _.get(source, 'metadata.name');
+  const namespace = _.get(source, 'metadata.namespace');
+  const targetResourceGroup = _.split(_.get(target, 'metadata.ownerReferences[0].apiVersion'), '/');
+  const targetResourceKind = _.get(target, 'metadata.ownerReferences[0].kind');
+  const targetResourceRefName = _.get(target, 'metadata.ownerReferences[0].name');
+  const sbrName = `${sourceName}-${source.kind}-${targetName}-${target.kind}`;
+
+  const serviceBindingRequest = {
+    apiVersion: 'apps.openshift.io/v1alpha1',
+    kind: 'ServiceBindingRequest',
+    metadata: {
+      name: sbrName,
+      namespace,
+    },
+    spec: {
+      applicationSelector: {
+        resourceRef: sourceName,
+        group: 'apps.openshift.io',
+        version: 'v1',
+        resource: modelFor(source.kind).plural,
+      },
+      backingServiceSelector: {
+        group: targetResourceGroup[0],
+        version: targetResourceGroup[1],
+        kind: targetResourceKind,
+        resourceRef: targetResourceRefName,
+      },
+    },
+  };
+
+  return k8sCreate(ServiceBindingRequestModel, serviceBindingRequest);
+};
+
+export const removeServiceBinding = (
+  source: K8sResourceKind,
+  target: K8sResourceKind,
+  sbrs: K8sResourceKind[],
+): Promise<any> => {
+  if (!source || !target || source === target) {
+    return Promise.reject();
+  }
+
+  const targetResourceKind = _.get(target, 'metadata.ownerReferences[0].kind');
+  const targetResourceRefName = _.get(target, 'metadata.ownerReferences[0].name');
+  const removeSbr = _.find(sbrs, (sbr) => {
```

Comment:
This can be avoided if you store the SBR in the edge data.

Reply:
Done.